### PR TITLE
feat: matomo wallets events

### PR DESCRIPTION
**Pull request by solidovic**
## Description

Add matomo wallets events via https://www.npmjs.com/package/reef-knot

### CODE REVIEW

**Review by kozlovzxc-at-lido — APPROVED** [review]

File: config/matomoWalletsEvents.ts [hunk 1]
```diff
@@ -0,0 +1,330 @@
...
+    'Click ZenGo wallet',
+    'eth_widget_click_zengo',
+  ],
+};
+
+export const walletsMetrics: WalletsMetrics = {
+  events: {
+    click: {
+      handlers: {
+        onClickAmbire: () => {
```
Comment (Jabher) [inline config/matomoWalletsEvents.ts:221]: better to introduce some local fn like this:
```typescript
const bindEvent = (event) => () => trackEvent(event)
```

Comment (solidovic) [reply config/matomoWalletsEvents.ts:223]: Looks like overheads.

I can't see the difference between this one:
```
const bindEvent = (event) => () => trackEvent(...event);

onClickAmbire: () => {
    bindEvent(MATOMO_WALLETS_EVENTS.onClickAmbire);
},
```

and second one:
```
onClickAmbire: () => {
    trackEvent(...MATOMO_WALLETS_EVENTS.onClickAmbire);
},
```

Or give more context, pls

File: config/matomoWalletsEvents.ts [hunk 2]
```diff
@@ -0,0 +1,330 @@
...
+  onClickZenGo = 'onClickZenGo',
+  onConnectZenGo = 'onConnectZenGo',
+}
+
+export const MATOMO_WALLETS_EVENTS: Record<
+  MATOMO_WALLETS_EVENTS_TYPES,
+  MatomoEventType
+> = {
+  [MATOMO_WALLETS_EVENTS_TYPES.onClickAmbire]: [
+    'Lido_Frontend_Template',
```
[the remark this answers is not in the document]
Comment (solidovic) [reply config/matomoWalletsEvents.ts]: done

File: MIGRATION.md [hunk 3]
```diff
@@ -206,8 +208,9 @@ const ETHERSCAN_SUBDOMAINS_BY_NETWORK = {
   [42]: 'kovan.',
 };
 
-export default getEtherscanLink(chain, hash) {
-  return `https://${ETHERSCAN_SUBDOMAINS_BY_NETWORK(chain)}/tx/${hash}`
+export default getEtherscanLink(chain, hash);
+{
```
Comment (alx-khramov) [inline MIGRATION.md:212]: This looks weird

File: config/matomoWalletsEvents.ts [hunk 4]
```diff
@@ -0,0 +1,332 @@
+import { MatomoEventType, trackEvent } from '@lidofinance/analytics-matomo';
+import { Metrics as WalletsMetrics } from 'reef-knot';
+
+const MATOMO_EVENT_CATEGORY = 'Lido_Frontend_Template';
```
Comment (alx-khramov) [inline config/matomoWalletsEvents.ts:4]: Maybe add a comment to make it clear, that this line should be changed to the name of Lido on X project?

File: MIGRATION.md (hunk 3 above)
Comment (solidovic) [reply MIGRATION.md:212]: thanks you, done :)

File: config/matomoWalletsEvents.ts (hunk 4 above)
Comment (solidovic) [reply config/matomoWalletsEvents.ts:4]: Good idea 👍

File: config/matomoWalletsEvents.ts [hunk 5]
```diff
@@ -1,8 +1,16 @@
 import { MatomoEventType, trackEvent } from '@lidofinance/analytics-matomo';
 import { Metrics as WalletsMetrics } from 'reef-knot';
 
+// Should be change for Lido on X project!!!
```
Comment (alx-khramov) [inline config/matomoWalletsEvents.ts:4]: @solidovic 
That's not grammatically correct.
"Should be change**d**".
But I would write something like
"Please, change the string to be relevant to your project. For example: ..."

Comment (solidovic) [reply config/matomoWalletsEvents.ts]: done

**Review by alx-khramov — APPROVED** [review]
Looks good to me ✅

**Review by Jabher — APPROVED** [review]

**Pull request merged**